cranelift-frontend/src/frontend.rs

```diff
@@ -523,7 +523,10 @@ impl FunctionBuilder {
     /// **Note:** this function has to be called at the creation of the `Ebb` before adding
     /// instructions to it, otherwise this could interfere with SSA construction.
     pub fn append_ebb_param(&mut self, ebb: Ebb, ty: Type) -> Value {
-        debug_assert!(self.func_ctx.ebbs[ebb].pristine);
+        debug_assert!(
+            self.func_ctx.ebbs[ebb].pristine,
+            "It can't add EBB parameters after adding any instruction"
```

Comment:
nit: the "It" at the beginning is uncommon in this code base, maybe we could either just remove it, or use an explicit "You can't add ..."?

Reply:
Done!